diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 15ebd6f3fbf6ceb13bb01fadc1e251d4fdd1d1cf..13394cf1087af4ba6cd56828f5ee8c19c41c6f62 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -8206,10 +8206,12 @@ def CheckSettingsChanges(input_api, output_api):
     relevant_files_found = False
     registry_content = ''
     registry_full_path = input_api.os_path.join(input_api.PresubmitLocalPath(),
+                                                "chrome", "android",
                                                 registry_path)
 
     try:
-        registry_content = input_api.ReadFile(registry_full_path)
+        import gclient_utils
+        registry_content = gclient_utils.FileRead(registry_full_path, 'r')
     except IOError:
         # If the registry cannot be read (e.g., file has been moved), we default
         # to an empty string. It is better to skip the registration check than
